python/ray/train/v2/_internal/state/util.py

```diff
@@ -0,0 +1,31 @@
+import time
+
+from ray.train.v2._internal.state.schema import (
+    ActorStatus,
+    RunAttemptStatus,
+    RunStatus,
+    TrainRun,
+    TrainRunAttempt,
+)
+
+
+def update_train_run_aborted(run: TrainRun) -> None:
+    run.status = RunStatus.ABORTED
+    run.status_detail = None  # TODO: Add status detail.
```

Comment:
Should we add the status detail now?

@matthewdeng do you have some context on what you wanted originally?